src/provider/index.ts

```diff
@@ -190,6 +230,59 @@ export class Main {
         return storedNotification;
     }
 
+    /**
+     * Dispatches, defers or discards a notification event
+     * Each event will be dispatched directly to the client application (and each one of its registered listener for the specific event)
+     * if there is at least one online client window is listening for the specific event.
+     * Event dispatch will be deferred if there is at least one listener for the specific application is registered for the specific event
+     * but there are no connected listeners at the time the event occurs.
+     * Events will be discarded if there are no client listeners registered for the specific app or all the registered listeners for the app
```

Comment:
Fix for clarity - "Events will be discarded if no client listeners have ever been registered for the specific event, or all the client listeners for the event have been unregistered."?